examples/src/embeddings/convex/cache_backed_convex.ts

```diff
@@ -0,0 +1,56 @@
+"use node";
+
+import { v } from "convex/values";
+import { TextLoader } from "langchain/document_loaders/fs/text";
+import { CacheBackedEmbeddings } from "langchain/embeddings/cache_backed";
+import { OpenAIEmbeddings } from "langchain/embeddings/openai";
+import { ConvexKVStore } from "langchain/storage/convex";
+import { RecursiveCharacterTextSplitter } from "langchain/text_splitter";
+import { ConvexVectorStore } from "langchain/vectorstores/convex";
+import { action } from "./_generated/server.js";
+
+export const ask = action({
+  args: {},
+  handler: async (ctx) => {
```

Comment:
I assume this does some env var fetching/populating behind the scenes?

Reply:
This code runs on the configured Convex server (the `npx convex dev` pushes the code to the server).